tests/test_operation.py

```diff
@@ -211,6 +291,29 @@ def test_has_matrix_false_concrete_template(self):
         op = qml.StronglyEntanglingLayers(params, wires=range(2))
         assert not op.has_matrix
 
+    @pytest.mark.tf
+    @pytest.mark.parametrize("jit_compile", [True, False])
+    def test_tf_function(self, jit_compile):
+        """Tests using tf.function with an operation works with and without
+        just in time (JIT) compilation."""
```

Comment:
Maybe I missed it, but are there torch/jax/jax jit tests? :)

Reply:
Will look into this. I'm confused with some tests structure.